src/blocks/weather.rs

```diff
@@ -138,6 +147,29 @@ impl Weather {
                     }
                 };
 
+                // Convert wind direction in azimuth degrees to abbreviation names
```

Comment:
The more idiomatic pattern here is to `match` on the range of values, e.g.
```rust
match direction {
  23..67 => "NE".to_string(),
  ...
  293..337 => "NW".to_string(),
  _ => "N".to_string(),
}
```

Reply:
Changed it, tanks for the hint!